UserService/src/main/java/org/energygrid/east/userservice/controller/UserController.java

```diff
@@ -1,16 +1,20 @@
 package org.energygrid.east.userservice.controller;
 
-import org.energygrid.east.userservice.errormessages.DuplicatedNameException;
 import org.energygrid.east.userservice.model.dto.UserDTO;
 import org.energygrid.east.userservice.model.fromFrontend.User;
 import org.energygrid.east.userservice.model.viewmodel.UserViewModel;
+import org.energygrid.east.userservice.rabbit.RabbitProducer;
 import org.energygrid.east.userservice.service.UserService;
 import org.modelmapper.ModelMapper;
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.http.ResponseEntity;
 import org.springframework.web.bind.annotation.*;
 
```

Comment:
```suggestion
import org.springframework.http.HttpStatus;
```
Added import for removed wildcard type